plugins/gin-gonic/gin/common.go

```diff
@@ -0,0 +1,226 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I take it you envision this being a common module of code shared by plugins. It might also be used in the built-in plugins included with the base library, right? There's a precedent in `sdk/resource/resourcekeys/const.go`, but I think these should be defined in the API not the SDK package tree.

Reply:
Yeah, but I'd like to have this code to be used by more before trying to put it into otel-go. For now it can be a shared module within otel-go-contrib.